### DISCUSSION TITLE
IP address preference

### DISCUSSION THREAD

**Discussion by PoneyClairDeLune** [v2fly/v2ray-core#1755] [Q&A]
Hello! Is there a way to configure servers to prefer using IPv4, and use IPv6 as a fallback? Or prefer one outbound address, and use others as fallbacks? If not, could you please consider adding it?

**Comment by AkinoKaede** [comment]
For Linux, you could build V2Ray with `CGO` and modify `/etc/gai.conf` to change the preference.

**Reply by PoneyClairDeLune** [reply]
It is kind of complicated to deploy to multiple servers... Will V2Ray be adding support for the preference in the future?
